compose/compose-file/index.md

```diff
@@ -882,6 +882,81 @@ port (a random host port will be chosen).
      - "127.0.0.1:5000-5010:5000-5010"
      - "6060:6060/udp"
 
+### secrets
+
+After you
+[define or reference secrets in the top-level `secrets` configuration](#secrets-configuration-reference),
+grant access to secrets on a per-service basis using the per-service `secrets`
+configuration. Two different syntax variants are supported.
```

Comment:
Maybe this is a minor nit pick, but it kind of bothers me that this reads like narrative documentation, when it's supposed to be reference docs.

So it might say something like:

"Secrets allows you to define or reference..."